test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -1283,6 +1283,54 @@ public void FindPage_LooksForPages_UsesRouteValuesAsFallback()
             Assert.Equal(expected, result.SearchedLocations);
         }
 
+        [Fact]
+        public void CreateCacheResult_LogsPrecompiledViewFound()
+        {
+            // Arrange
+            var sink = new TestSink();
+            var loggerFactory = new TestLoggerFactory(sink, enabled: true);
+
+            var relativePath = "/Views/Foo/details.cshtml";
+            var page = new Mock<IRazorPage>(MockBehavior.Strict);
+            var pageFactory = new Mock<IRazorPageFactoryProvider>();
+            pageFactory
+                .Setup(p => p.CreateFactory(relativePath))
+                .Returns(new RazorPageFactoryResult(() => page.Object, new IChangeToken[0], isPrecompiledView: true))
```

Comment:
`.Returns(new RazorPageFactoryResult(() => Mock.Of<IRazorPage>(), new , new IChangeToken[0], isPrecompiledView: true)` and remove `var page ...`